packages/fuels-core/src/errors.rs

```diff
@@ -75,13 +74,25 @@ impl From<ParseError> for Error {
     }
 }
 
-impl From<InvalidOutputType> for Error {
-    fn from(err: InvalidOutputType) -> Error {
-        Error::ContractCallError(err.0, vec![])
-    }
-}
 impl From<std::io::Error> for Error {
     fn from(err: std::io::Error) -> Error {
         Error::ContractCallError(err.to_string(), vec![])
     }
 }
+
+#[derive(Debug)]
+pub struct InstantiationError(pub String);
```

Comment:
Nice new error, made things much clearer on the error handling side of this process!